### ISSUE TITLE
View "graph overview" of blocks.

### ISSUE THREAD

**Issue by levinelson** [Roam-Research/issues#531]
**Describe the problem**

Currently, from what I can tell, the graph overview only connects _pages_.

**Feature implementation**

For the way I use Roam, I largely write "slip box" style notes as blocks under each day, and then directly link those blocks to each other. I'd love to see a graph overview of how my _blocks_ are directly connected to each other somehow.

**Comment by vkazartsev** [comment]
That is actually a great idea for those trained to write top-down pyramid style (not many are, sadly).

I've been looking for a tool to implement this for some time, even considering a custom-configured wiki engine (amazing what those can do, not too user-friendly, though).

The one conceptual issue to resolve here is compact representation for blocks: unlike pages, blocks have no title 🤔

**Comment by levinelson** [comment]
Yeah, true. I think even representing the first X number of characters could work for this sort of thing, and then maybe represent the whole block in a sidebar when clicked?
